src/main/java/edu/harvard/iq/dataverse/license/TermsOfUse.java

```diff
@@ -0,0 +1,159 @@
+package edu.harvard.iq.dataverse.license;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import edu.harvard.iq.dataverse.FileMetadata;
+
+/**
+ * Entity describing on what terms
+ * file can be used by app users.
+ * 
+ * @author madryk
+ */
+@Entity
+public class TermsOfUse implements Serializable {
```

Comment:
Hmm since we already have OtherTermsOfUse tab and that could be confusing that there are Licenses and TermsOfUse. Maybe FileTermsOfUse/ FileBasedTermsOfUse?

Reply:
ok, went for FileTermsOfUse